images/builder-protobuf/Dockerfile

```diff
@@ -6,13 +6,15 @@ ENV GOLANGCI_LINT_CACHE $GOPATH/cache
 ENV GOCACHE $GOPATH/cache
 
 SHELL ["/bin/ash","-c"]
+WORKDIR /usr/work
+LABEL org.opencontainers.image.source="https://github.com/dyrector-io/dyrectorio/tree/develop/images/builder-protobuf"
```

Comment:
You are not missing the Dockerfile from the end of the PATH?

Reply:
Not really, as the directory is the whole context of the build. The `docker build` command will accept a directory as parameter and search a Dockerfile in that directory. @nandor-magyar Is this correct this way?

Reply:
Yes, this is the correct way, given the spec. This has nothing to do with the docker build command though, this is only for documentation purposes (afaik).